apps/prairielearn/src/lib/workspace.js

```diff
@@ -451,6 +486,19 @@ module.exports = {
       }
     }
 
+    issues.forEach(async (issue) => {
+      await sqldb.callAsync('issues_insert_for_variant', [
```

Comment:
Rather than directly calling this sproc, use `insertIssue(...)` from `lib/issues` instead.

Reply:
Done in a83dae3.